onnx/common/ir.h

```diff
@@ -767,6 +767,56 @@ struct Node : public Attributes<Node> {
   }
 };
 
+// A class with the same properties as OperatorSetIdProto, but without protobuf
+// overhead, resulting in a simpler and more readable workflow.
+class OpSetID final {
+  private:
+    std::string domain_;
+    int64_t version_;
+
+  public:
+    explicit OpSetID(const ONNX_NAMESPACE::OperatorSetIdProto& proto)
+      :domain_(proto.domain()), version_(proto.version()) {}
+
+    // Default Domain Constructor
+    explicit OpSetID(const int64_t& version)
```

Comment:
no need for reference because reference is 64bit and int64_t is also just 64bit - i.e. copying an int64_t is not more expensive than copying the reference. Then, the indirection of a reference here introduces a small performance hit.